src/settings/UserListView.ts

```diff
@@ -128,56 +126,56 @@ export class UserListView implements UpdatableSettingsViewer {
 		this.list.loadInitial()
 		const searchBar = neverNull(header.searchBar)
 
-		this._listId.getAsync().then(listId => {
+		this.listId.getAsync().then(listId => {
 			searchBar.setGroupInfoRestrictionListId(listId)
 		})
 
-		this._searchResultStreamDependency = searchBar.lastSelectedGroupInfoResult.map(groupInfo => {
-			if (this._listId.isLoaded() && this._listId.getSync() === groupInfo._id[0]) {
+		this.searchResultStreamDependency = searchBar.lastSelectedGroupInfoResult.map(groupInfo => {
+			if (this.listId.isLoaded() && this.listId.getSync() === groupInfo._id[0]) {
 				this.list.scrollToIdAndSelect(groupInfo._id[1])
 			}
 		})
 
 		this.onremove = () => {
-			if (this._searchResultStreamDependency) {
-				this._searchResultStreamDependency.end(true)
+			if (this.searchResultStreamDependency) {
+				this.searchResultStreamDependency.end(true)
 			}
 		}
 	}
 
-	async _loadAdmins(): Promise<void> {
+	private async loadAdmins(): Promise<void> {
```

Comment:
If it's not async, let's make it sync?

Reply:
it does async stuff, we just don't await it. I personally prefer leaving it transparent